SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php

```diff
@@ -158,4 +169,14 @@ public function process(File $phpcsFile, $docCommentOpenPointer): void
 		}
 	}
 
+	/**
+	 * @return list<string>
+	 */
+	private function getIgnoredAnnotationNames(): array
+	{
+		$this->ignoredAnnotationNames = SniffSettingsHelper::normalizeArray($this->ignoredAnnotationNames);
```

Comment:
Please normalize the array in `process` method, see https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Classes/AbstractPropertyConstantAndEnumCaseSpacing.php#L58-L61